crypto/sessionkey.go

```diff
@@ -98,12 +98,16 @@ func newSessionKeyFromEncrypted(ek *packet.EncryptedKey) (*SessionKey, error) {
 		return nil, fmt.Errorf("gopenpgp: unsupported cipher function: %v", ek.CipherFunc)
 	}
 
-	symmetricKey := &SessionKey{
+	sk := &SessionKey{
 		Key:  ek.Key,
 		Algo: algo,
 	}
 
-	return symmetricKey, nil
+	if err := sk.checkSize(); err != nil {
+		return nil, errors.Wrap(err, "gopenpgp: unable to decode session key")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "gopenpgp: unable to decrypt session key")
```
Since this function is only called from `DecryptSessionKey` and I think it doesn't really make sense in any other context